flag_int_slice.go

```diff
@@ -44,6 +44,18 @@ func (i *IntSlice) Set(value string) error {
 		return nil
 	}
 
+	if strings.Contains(value, ",") {
```

Comment:
Nit: Same note as above—I don't believe we have to special case this.

Reply:
Fixed this too